src/routes/Router/Router.tsx

```diff
@@ -19,36 +22,52 @@ export const Router = createBrowserRouter(
       errorElement={<NotFoundSite />}
     >
       <Route
-        path={RouterPath.OfferedApiDelegations + '/' + RouterPath.Overview}
+        path={ApiDelegationPath.OfferedApiDelegations}
```

Comment:
Is this a path? Just the OfferedApiDelegations, without the overview?

Reply:
Its a way of structuring different child routes to parent routes to avoid errors and duplication of code. See the new Router I'll soon push for more like what it should be like

Reply:
With that change we also got hot-reload on direct elements 🥳 
(It's probably the correct way of doing it)

Reply:
Aaaahh, I didn't realize that they were parent and child with the diff being all over the place
But that makes sense! 😁